spec/policies/request_for_comment_policy_spec.rb

```diff
@@ -46,6 +52,30 @@
         end
       end
     end
+
+    permissions(:report?) do
+      context 'when report emails are configured' do
+        let(:reports_enabled) { true }
+
+        it 'allows anyone to report RfCs expect the author' do
+          %i[admin external_user teacher].each do |factory_name|
+            expect(policy).to permit(create(factory_name), rfc)
+          end
+        end
+
+        it 'blocks authors from reporting their content' do
+          expect(policy).not_to permit(rfc.author, rfc)
+        end
```

Comment:
I thought about streamlining that. What do you think?
```suggestion
        it 'allows anyone to report RfCs expect the author' do
          %i[admin external_user teacher].each do |factory_name|
            expect(policy).to permit(create(factory_name), rfc)
          end
          expect(policy).not_to permit(rfc.author, rfc)
        end
```